daml_dit_api/main/main.py

```diff
@@ -50,28 +68,29 @@ def create_network(url: str) -> 'Network':
     return network
 
 
-def load_integration_spec(config: 'Configuration') -> 'IntegrationRuntimeSpec':
-    metadata_path = Path(config.integration_metadata_path)
-
-    if metadata_path.exists():
-        LOG.info('Loading integration metadata from: %r', metadata_path)
-
-        return from_dict(
-            data_class=IntegrationRuntimeSpec,
-            data=yaml.safe_load(metadata_path.read_bytes()))
-    else:
-        raise Exception('No metadata file found at: ' + repr(metadata_path))
-
+async def run_dazl_networks(*networks: 'Network'):
+    """
+    Run the dazl networks, and make sure that fatal dazl errors terminate the application.
+    """
+    LOG.info('Starting dazl network(s)...')
+    # noinspection PyBroadException
+    try:
+        futures = [ensure_future(n.aio_run()) for n in networks]
+        await gather(*futures)
+    except:  # noqa
+        FAIL('The main dazl coroutine died with an exception')
```

Comment:
It's probably not completely related to this PR, but if there is only ever one `Network` being connected to, this whole thing should probably just be simplified to `await network.aio_run()` instead.

This originally surfaced in `dabl-services` when we used to talk to both child ledgers and the admin ledger at the same time, and I'm not sure there's really a need for it any more in any of our applications.

Reply:
Thanks... I'd actually meant to include a fix for that in this.

Reply:
Fixed.